frontend/components/forms/__tests__/Input.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from 'lib/jest'
+import { Input } from '../Input'
+
+describe('<Input />', () => {
+  it('shows the passed label', () => {
```

Comment:
Same comment as the label test. Might be worth testing if a mocked `onChange` function is called